### PR TITLE
AAE-19479 Register a new primitive type for decimal

### PR DESCRIPTION
**Pull request by tom-dal**
Issu link: AAE-19479

### CODE REVIEW

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/resources/schema/process-extensions-schema.json [hunk 1]
```diff
@@ -281,6 +282,37 @@
...
         },
+        {
+          "if": {
+            "properties": {
+              "type": {
+                "const": "bigdecimal"
+              }
+            }
+          },
+          "then": {
```
Comment (jesty) [inline activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/resources/schema/process-extensions-schema.json:293]: how did you validate the type and the pattern in the anyOf?

Comment (tom-dal) [reply activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/resources/schema/process-extensions-schema.json:293]: I can add a validation pattern in the first element of the anyOf but I don't know if it's needed (please look at the "integer" case)

**Comment by wojciech-piotrowiak** [comment]
@tom-dal Can you wait a few days with the merge?
We are in the refactor process, the whole modeling code will be moved to modeling module (epic link https://alfresco.atlassian.net/browse/AAE-19019)

**Comment by tom-dal** [comment]
@wojciech-piotrowiak sure

**Comment by wojciech-piotrowiak** [comment]
@tom-dal 
At this moment we copied most of the modeling code from activiti to modeling.
Please place the code there once you finish (Assume acceptance tests need to be green)

Modeling code will be removed from activiti soon, once we will finish with acceptance tests coverage.

**Comment by tom-dal** [comment]
@wojciech-piotrowiak Thanks for the update! I'll give it a check.

**Pull request closed without merging**